spec/app/models/mdm/web_page_spec.rb

```diff
@@ -3,6 +3,48 @@
 
   context 'associations' do
     it { is_expected.to belong_to(:web_site).class_name('Mdm::WebSite') }
+
+    context 'serialized cookie attribute' do
+      let(:valid_cookie) do
+        {
+          name: 'test name',
+          value: 'test value'
+        }
+      end
+
+      let(:invalid_cookie) do
```

Comment:
@limhoff-r7 I'm not sure how to test if this will cause breakage somewhere in app. Yes it looks like the cookie would still be unusable, meaning we could sanitize the cookie based on format. Import the zip does work, it just creates the (unusable) cookie attribute. Creating a true solution to this may be beyond my scope but I'm open to suggestions